features/util.go

```diff
@@ -109,6 +119,19 @@ func (b *featureGateBuilder) register() (configv1.FeatureGateName, error) {
 	if len(b.owningProduct) == 0 {
 		return "", fmt.Errorf("missing owningProduct")
 	}
+	_, enhancementPRErr := url.Parse(b.enhancementPRURL)
+	switch {
+	case b.enhancementPRURL == "FeatureGate predates 4.18":
+
+	case len(b.enhancementPRURL) == 0:
+		return "", fmt.Errorf("FeatureGate/%s is missing an enhancementPR with GA Graduation Criteria like https://github.com/openshift/enhancements/pull/#### or https://github.com/kubernetes/enhancements/issues/####", b.name)
```

Comment:
Not necessarily relevant here, but when an error is returned from `register`, we panic with the error. In this case you've added the name to the format string, but most exits from this function do not, should we update the panic to `fmt.Sprintf("FeatureGate/%s: %v", b.name, err)` so that all panics are prefixed with the gate name?

Reply:
Additionally, we could have a few Feature Gate that has both upstream and downstream enhancements.

Looks like the current validation can cover it if it's like `https://github.com/openshift/enhancements/pull/xxxx,https://github.com/kubernetes/enhancements/issues/xxxxx` yet better if the format could be standardized in such cases.